.github/workflows/release.yml

```
@@ -0,0 +1,16 @@
+name: Syrupy Release
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: cycjimmy/semantic-release-action@v2
+        with:
+          branch: master
+          dry_run: false
+          extra_plugins: |
+            @semantic-release/changelog
+            @semantic-release/git
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

@noahnu might need to add here a custom personal token secret with permission to push changelog updates to github, if the default `GITHUB_TOKEN` is restricted.
```suggestion
          GH_TOKEN: ${{ secrets.GH_TOKEN }}
```